src/bento/components/amp-jwplayer/1.0/component.type.js

```diff
@@ -0,0 +1,25 @@
+/** @externs */
+
+/** @const */
+var BentoJwplayerDef = {};
+
+/**
```

Comment:
Question for Bento developers: are there any attributes that do not work on the CE standalone component and only work on the AMP component? If so, we'll need to create two type files